packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -131,7 +130,7 @@ export const NftWidget: FC<NftWidgetProps> = ({
         <NftInfoItem
           size={size}
           label="Buy now"
-          disableSecondary={!JOY_PRICE_SERVICE_URL}
+          disableSecondary={convertHapiToUSD(buyNowPrice) === null}
```

Comment:
Could we just store result of `convertHapiToUSD(buyNowPrice)` in a variable? We do it 3 separate times now